examples/image-to-text/README.md

```diff
@@ -131,6 +132,16 @@ python3 run_pipeline.py \
     --sdp_on_bf16
 ```
 
+To run inference with THUDM/glm-4v-9b, use the following command (Note that you need to set the environment variable `GLM=4v` to distinguish between glm4v and chatglm, as these models are customized and share the same model type named "chatglm"):
+```bash
+GLM=4v python3 run_pipeline.py \
+    --model_name_or_path THUDM/glm-4v-9b \
+    --use_hpu_graphs \
+    --bf16 \
+    --sdp_on_bf16 \
+    --use_flash_attention \
+    --use_kv_cache
+
 ### Inference with FP8
```

Comment:
no need to add in readme

Reply:
> no need to add in readme

For glm4v, it needs a environment variable GLM=4v to set before running the inference cmd, this is because glm4v shares the same model_type as chatglm defined in their own config.json, and this env var helps to ensure the correct model is registered. I think it necessary to remind user in README of how to run glm4v inference.